### PR TITLE
creating veterans page

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/z6edtqUf/3477-create-veterans-page

### Context

Creating a new veteran's page to sit under 'funding and support'.

### CODE REVIEW

**Comment by EllieNodder** [comment]
@gemmadallmandfe I know you commented in the doc that the title doesn't quite work. All the titles in this section for if you're disabled/if you're a parent or carer/if you come from outside England start in the same way: 'Support training to teach if you're ...'. 

If we change one, we should probably change the others to match. Do you think '**Get** support training to teach if you're ...' works better? Or something else?

(@GDW7 good to get your opinion on this and the rest of the page, too)

**Comment by gemmadallmandfe** [comment]
> 

Yes, I think if one changes, they all need to - but I'll leave it to you two to work out if/what!

This came up in Keisha and Steph's card sort - people didn't seem to understand the titles as I recall - so worth speaking to them to about this

**Comment by gemmadallmandfe** [comment]
@EllieNodder You might want to leave any h1 change for now though - this can be picked up with wider changes to the Funding and support section based on the Why teach UR insights

ie publish this veterans page now (when approved by pipeline) with the same 'Support training to teach....' heading as the others, and address a possible change in h1s later/separately?

**Comment by EllieNodder** [comment]
> @EllieNodder You might want to leave any h1 change for now though - this can be picked up with wider changes to the Funding and support section based on the Why teach UR insights
> 
> ie publish this veterans page now (when approved by pipeline) with the same 'Support training to teach....' heading as the others, and address a possible change in h1s later/separately?

That makes sense to me

File: app/views/content/funding-and-support/if-youre-a-veteran.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+As a veteran, you can bring a unique perspective to teaching by using your organisation, leadership and management skills in the classroom.
+
+There are full and part-time teacher training courses available.
+
+## Undergraduate bursary for veterans
+
+You may be eligible for a tax-free undergraduate bursary of £40,000 if you’re:
+
+* a veteran
```
Comment (GDW7) [inline app/views/content/funding-and-support/if-youre-a-veteran.md:36]: Hi @EllieNodder with this bullet list do you know if any of the conditions make you eligible, or are they all required? It's not clear to me from reading it at the moment.

File: app/views/content/funding-and-support/if-youre-a-veteran.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* training to teach secondary biology, physics, chemistry, computing, maths or languages
+* doing an undergraduate degree leading to [qualified teacher status (QTS)](/what-is-qts) in England
+
+Doing an undergraduate degree leading to QTS means that you would be qualified to teach in England.
+
+You do not need to apply for a bursary. Your teacher training provider will determine whether you are eligible and will then pay you in monthly instalments across the final 2 years of your course.
```
Comment (GDW7) [inline app/views/content/funding-and-support/if-youre-a-veteran.md:42]: @EllieNodder is this sentence saying you don't need to apply for a bursary to get one? Or you don't need to apply for a bursary to train to teach?

**Comment by GDW7** [comment]
Hi @EllieNodder I think this looks good. Just a couple of small comments above. 

Plus a thought on the category page card: do we want to consider adding 'armed forces' to the copy there somewhere so that it's obvious who it is and isn't for. For people not from the armed forces coming to the category page, will they immediately know what 'If you're a veteran' means in this context and be able to decide it's not for them without clicking through?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

File: app/views/content/funding-and-support/if-youre-a-veteran.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+### Get an adviser
+
+You can get free dedicated support from a teacher training adviser and talk to someone just like you who has left the armed forces and gone into teaching.
+
+As a veteran, you can get an adviser with or without a degree.
+
+Call us on **0800 389 2500** to get an adviser.
+
+Our telephone service is available Monday to Friday between 8:30am and 5:30pm
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/if-youre-a-veteran.md:68]:
```suggestion
Our telephone service is available Monday to Friday between 8:30am and 5:30pm.
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**